.github/workflows/phpunit.yml

```diff
@@ -9,7 +9,7 @@ jobs:
   phpunit:
     runs-on: ubuntu-latest
     container:
-      image: kirschbaumdevelopment/laravel-test-runner:7.3
+      image: kirschbaumdevelopment/laravel-test-runner:8.1
```

Comment:
Whoah, good catch!

Reply:
Had my [eye](https://github.com/GCTC-NTGC/gc-digital-talent/issues/3839#issue-1360573210) on this one for a few months!